Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -395,355 +395,677 @@ public enum Language
     {
 #pragma warning disable 1591
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Abap")]
+        [Parameter(Value = "abap")]
         Abap,
+
         [Parameter(Value = "ActionScript")]
         ActionScript,
+        
+        [Parameter(Value = "ada")]
         Ada,
+        
+        [Parameter(Value = "apex")]
         Apex,
+        
         [Parameter(Value = "AppleScript")]
```

Comment:
Pinned this back to https://github.com/octokit/octokit.net/commit/e01da11af5c821225995711ede0a74c37a7bd2ea. Looks like the idea of introducing these was *"let's not let the serialization strategy transform them to all-lowercase"*.

It doesn't make a difference as [https://api.github.com/search/repositories?q=github+language:applescript](https://api.github.com/search/repositories?q=github+language:applescript) fetches the same results as [https://api.github.com/search/repositories?q=github+language:AppleScript](https://api.github.com/search/repositories?q=github+language:AppleScript). As a result, I'm leaning towards changing the ones that are equivalent to the member name to be all lowercase for consistency.

A possible official source for languages could be https://github.com/github/linguist/blob/master/lib/linguist/languages.yml. It matches a few languages (C# and C++) that we aliased (csharp and cpp)